src/test/shared/utilities/cliUtils.test.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+import { Logger } from '../../../shared/logger'
+import { makeLogger } from '../../../shared/logger/activation'
+import { WinstonToolkitLogger } from '../../../shared/logger/winstonToolkitLogger'
+import { installCli } from '../../../shared/utilities/cliUtils'
+import { FakeWindow } from '../vscode/fakeWindow'
+import { TestSettingsConfiguration } from '../../utilities/testSettingsConfiguration'
+
+describe('cliUtils', async function () {
+    let tempFolder: string
+    let testLogger: Logger | undefined
+
+    const settingsConfig = new TestSettingsConfiguration()
+    // confirms installation confirmation prompt
+    const acceptInstallWindow = new FakeWindow({
+        message: {
+            informationSelection: 'Install',
+        },
+    })
+
+    before(async function () {
+        settingsConfig.writeSetting('aws.dev.forceInstallTools', true)
+        tempFolder = await makeTemporaryToolkitFolder()
+        testLogger = makeLogger({ staticLogLevel: 'debug', logPaths: [path.join(tempFolder, 'log.txt')] })
```

Comment:
`testLogger` doesn't do anything?

Reply:
I'm kinda thinking that these were around for the sake of running the tests as `.only` during testing (since the code has a fair amount of getLogger() calls). Tested without, confirmed works, will remove.

Reply:
May need to figure out some boundaries between test code and global state, though it's difficult to separate the two. Potentially have test setup instantiate/reset instances, and trying to create a new instance just throws. We could also just inject a fake global context into each describe block, that way tests don't need to manage construct and tear-down logic.